kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -2,10 +2,10 @@
 
   <breadcrumbs v-if="breadcrumbs"></breadcrumbs>
 
-  <card-grid header="Topics" v-if="topics.length">
+  <card-grid header="Topics" v-if="subtopics.length">
     <topic-card
-      v-for="topic in topics"
-      :id="topic.pk"
+      v-for="topic in subtopics"
+      :id="topic.id"
```

Comment:
the vuex state no longer corresponds directly to API attributes. The mappings occur in actions.js in:

* `_crumbState`
* `_topicState`
* `_contentState`